client/src/Components/Sidebar.jsx

```diff
@@ -14,11 +13,19 @@ import SignOutIcon from "@mui/icons-material/ExitToApp";
 import Avatar from "@mui/material/Avatar";
 import { useNavigate } from "react-router-dom";
 import useAuth from "../customHooks/useAuth";
+import { useContext, useEffect, useState } from "react";
+import { AppContext } from "../App";
 
 export default function TemporaryDrawer() {
 	const navigate = useNavigate();
 	const { handleSignOut, handleDeleteProfile } = useAuth();
-	const [right, setRight] = React.useState(false);
+	const { user } = useContext(AppContext);
+	const [right, setRight] = useState(false);
+	const [picture, setPicture] = useState("");
```

Comment:
Why are we using an empty string as the initial state. Why not `null`

Reply:
true